hedera-node/hedera-app/src/main/java/com/hedera/node/app/config/converter/CongestionMultipliersConverter.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.config.converter;
+
+import com.hedera.node.app.service.mono.fees.calculation.CongestionMultipliers;
+import com.swirlds.config.api.converter.ConfigConverter;
+
+/**
+ * Config api {@link ConfigConverter} implementation for the type {@link CongestionMultipliers}.
+ */
+public class CongestionMultipliersConverter implements ConfigConverter<CongestionMultipliers> {
+
+    @Override
+    public CongestionMultipliers convert(final String value) throws IllegalArgumentException, NullPointerException {
+        if (value == null) {
+            throw new NullPointerException("null can not be converted");
+        }
+        return CongestionMultipliers.from(value);
```

Comment:
Since you defined `ConfigConverter` as a `FunctionalInterface` even explicitly, I wonder if it makes sense to take advantage of that in cases like this. We could get rid of several classes.

Reply:
Like the idea. I just copied the original conversion code from the mono services module. Fine for change this laters once the PR is merged.